doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.6.99-cs2_to_2_7_environment.rst

```diff
@@ -0,0 +1,349 @@
+
+#. Run the script:
+
+   * Linux and Mac OS::
+
+        JLinkExe -CommanderScript nrf54h20es_trim_adjust.jlink
+
+   * Windows::
+
+        jlink.exe -CommanderScript nrf54h20es_trim_adjust.jlink
+
+.. _migration_cs2_to_2_7_env_lcsrot:
+
+.. rst-class:: numbered-step
+
+Transitioning the nRF54H20 SoC to RoT
+=====================================
+
+Set the device to Root of Trust using the following command::
+
+      nrfutil device x-adac-lcs-change --life-cycle rot --serial-number <serial_number>
+
+#. Verify again the current lifecycle state of the nRF54H20::
+
+      nrfutil device x-adac-discovery --serial-number <serial_number>
+
+   The output will look similar to the following::
+
+      *serial_number*
+      adac_auth_version     1.0
+      vendor_id             Nordic VLSI ASA
+      soc_class             0x00005420
+      soc_id                [e6, 6f, 21, b6, dc, be, 11, ee, e5, 03, 6f, fe, 4d, 7b, 2e, 07]
+      hw_permissions_fixed  [00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      hw_permissions_mask   [01, 00, 00, 00, 87, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00, 00]
+      psa_lifecycle         LIFECYCLE_ROT (0x2000)
+      sda_id                0x01
+      secrom_revision       0xad3b3cd0
+      sysrom_revision       0xebc8f190
+      token_formats         [TokenAdac]
+      cert_formats          [CertAdac]
+      cryptosystems         [Ed25519Sha512]
+      Additional TLVs:
+      TargetIdentity: [ff, ff, ff, ff, ff, ff, ff, ff]
+
+   The lifecycle state (``psa_lifecycle``) is now correctly set to *Root of Trust* (``LIFECYCLE_ROT (0x2000)``)
+
+#. After the LCS transition, reset the device::
+
+      nrfutil device reset --reset-kind RESET_PIN --serial-number <serial_number>
+
+nRF54H20 DK firmware bundle update
+**********************************
+
+.. caution::
+   Follow these steps only if your device is in lifecycle state ``RoT``.
+   For more information, see `Verify the LCS of the device`_.
+
+If your device is in ``RoT``, you can update to the newest release of the firmware bundle as follows:
+
+1. Download the `nRF54H20 firmware bundle v0.5.0`_.
+#. Move the :file:`ZIP` bundle to a folder of your choice and unzip the bundle.
+#. Purge all the domains::
+
+      nrfutil device x-boot-mode-set --boot-mode safe --traits jlink
+      nrfutil device erase --all --traits jlink --log-level trace --core Application
+      nrfutil device erase --all --traits jlink --log-level trace --core Network
+      nrfutil device x-boot-mode-set --boot-mode normal --traits jlink
+
+#. Program the SDFW update::
+
+      nrfutil device program --options chip_erase_mode=ERASE_NONE --firmware uslot_download.hex --core Application --serial-number <serial_number>
+
+#. Instruct SDFW to instruct SDROM to apply the update::
+
+      nrfutil device x-ssf-sdfw-update --traits jlink --tbs-addr 0x0e100000 --dl-max 0x4b000 --dl-addr-fw 0x0e100160 --dl-addr-pk 0x0e100100 --dl-addr-signature 0x0e100120 --log-level debug
```

Comment:
```suggestion
      nrfutil device x-ssf-sdfw-update --tbs-addr 0x0e100000 --dl-max 0x4b000 --dl-addr-fw 0x0e100160 --dl-addr-pk 0x0e100100 --dl-addr-signature 0x0e100120 --log-level debug --serial-number <serial_number>
```

Reply:
Removed the entire section as outdated, as it is not possible to update the firmware bundle when the DK has LCS RoT

Reply:
That is news to me, I have done this update myself in LCS RoT. My understanding is that `nrfutil device x-ssf-sdfw-update` should work to update a device which has been provisioned with nrf54h20_soc_binaries_v0.3.x.zip. With newer versions of the bundle, suit can be used.

Reply:
@Tiwalun @ns-tolu Can you clarify this? We can eventual add these steps back in a separate PR